src/authorizations/components/redesigned/ResourceAccordionHeader.tsx

```diff
@@ -65,35 +41,42 @@ export const ResourceAccordionHeader: FC<OwnProps> = props => {
         stretchToFitWidth={true}
         alignItems={AlignItems.Center}
         style={{textAlign: 'start'}}
-        onClick={(e: any) => {
-          e.stopPropagation()
-        }}
       >
         <FlexBox.Child basis={40} grow={8}>
           <InputLabel size={ComponentSize.Medium}>{title}</InputLabel>
         </FlexBox.Child>
-        <FlexBox.Child grow={1}>
+        <FlexBox.Child
+          grow={1}
+          onClick={(e: any) => {
+            e.stopPropagation()
+          }}
```

Comment:
since this is used in three places, why not make a single function that does this that doesn't have to be created on every render loop - right now this same function gets created 3x per render, but it only really needs to be created once.

Also, can you call it `event` or `evt` instead of `e` please? 😁